file.css

```diff
@@ -0,0 +1,3 @@
+.header {
+  color: rgb(26, 26, 99);
+}
```

Comment:
Kindly follow the naming convention for CSS files. You can name the like 'style.css' instead of the file.css